### PR TITLE
Updating the readme file.

### PR DESCRIPTION
**Pull request by AnnRiera**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: [CONTRIBUTING.md](https://github.com/jarcodallo/nestjs-matic/blob/master/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes

Updating the readme file.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,7 +8,186 @@ NestJS-Matic
...
+
+### Zero configuration
+
+Import the `MaticModule` to your module and pass in it through your matic object options, just like that:
+
+```ts
+import { MaticModule } from 'nestjs-matic';
+
+@Module({
+  imports: [MaticModule.forRoot(MATIC_OPTIONS)],
```
Comment (0xslipk) [inline README.md:27]: if this is zero, configuration, you shouldn't pass any option.

File: README.md [hunk 2]
```diff
@@ -9,7 +8,186 @@ NestJS-Matic
...
+  ...
+})
+class MyModule {}
+```
+
+**NOTE:** we describe the [MATIC_OPTIONS](#configuration-params) down below.
```
Comment (0xslipk) [inline README.md]: I think here you need to put something like:  By default MaticModule will try to connect using the MaticPlama client.

File: README.md [hunk 3]
```diff
@@ -9,7 +8,186 @@ NestJS-Matic
...
+   * @see {@link https://github.com/ChainSafe/web3.js/tree/1.x/packages/web3-core}
+   * @see {@link https://www.npmjs.com/package/@walletconnect/ethereum-provider}
+   */
+  parentProvider?: string | HttpProvider | WalletConnectProvider;
+
+  /**
+   * Optional parameter for parentDefaultOptions, must be a Matic
+   * SendOptions type.
+   * If no parentDefaultOptions is provided, empty object is used.
+   * @see {@link https://github.com/maticnetwork/matic.js/blob/241d83873bb02501dd01243cd43e31265bc6ba0c/src/types/Common.ts#L3}
```
Comment (0xslipk) [inline README.md]: Use the link to the master branch https://github.com/maticnetwork/matic.js/blob/master/src/types/Common.ts#L3

File: README.md [hunk 4]
```diff
@@ -9,7 +8,186 @@ NestJS-Matic
...
+   */
+  parentDefaultOptions?: SendOptions;
+
+  /**
+   * Optional parameter for maticDefaultOptions, must be a Matic
+   * SendOptions type.
+   * If no parentDefaultOptions is provided, empty object is used.
+   * @see {@link https://github.com/maticnetwork/matic.js/blob/241d83873bb02501dd01243cd43e31265bc6ba0c/src/types/Common.ts#L3}
```
Comment (0xslipk) [inline README.md]: Use the link to the master branch https://github.com/maticnetwork/matic.js/blob/master/src/types/Common.ts#L3

File: README.md [hunk 5]
```diff
@@ -9,7 +8,186 @@ NestJS-Matic
...
+   */
+  maticDefaultOptions?: SendOptions;
+
+  /**
+   * Optional parameter for maticClient, must be a string.
+   * Take 'Plasma' or 'PoS' as possible values.
+   * If no maticClient is provided, 'PoS" is used.
```
Comment (0xslipk) [inline README.md]: The default is Plasma client

File: README.md (hunk 2 above)
Comment (AnnRiera) [reply README.md]: Ok, I will fix this out :+1:

**Review by 0xslipk — APPROVED** [review]

**Pull request merged**